modules/container/map.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package container
+
+// MapValues returns a slice of all values in a map.
+// TODO: remove this after Golang std lib has similar function but not golang.org/x/exp
+func MapValues[K comparable, V any](m map[K]V) []V {
```

Comment:
Either remove this one or
```go
modules/util/slice.go
58:func ValuesOfMap[K comparable, V any](m map[K]V) []V {
```

Reply:
833ec563792b2fa883dbd147df19bb09bef61277